src/ocean/math/Range.d

```diff
@@ -215,6 +215,8 @@ public struct Range ( T )
     {
         public istring toString()
         {
+            import ocean.core.TypeConvert: assumeUnique;
```

Comment:
This seems unrelated to the commit. Looks like you removed an unnecessary import here. But I don't see by which import it is actually covered.

Reply:
This was the only remaining use of `ocean.transition` in this module, which is the reason for making these changes. It will be removed from everywhere else in ocean in the next PR.